specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2019-03-01/application.json

```diff
@@ -0,0 +1,2199 @@
+      }
+    },
+    "OperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of operations supported by the Service Fabric resource provider.",
+          "items": {
+            "$ref": "#/definitions/OperationResult"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any.",
+          "readOnly": true
+        }
+      },
+      "description": "Describes the result of the request to list Service Fabric resource provider operations."
+    },
+    "OperationResult": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the operation."
+        },
+        "display": {
+          "$ref": "#/definitions/AvailableOperationDisplay",
+          "description": "The object that represents the operation."
+        },
+        "origin": {
+          "type": "string",
+          "description": "Origin result"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "Available operation list result"
+    },
+    "PartitionScheme": {
+      "type": "string",
+      "description": "Enumerates the ways that a service can be partitioned.",
+      "enum": [
+        "Invalid",
+        "Singleton",
+        "UniformInt64Range",
+        "Named"
+      ],
+      "x-ms-enum": {
+        "name": "PartitionScheme",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Invalid",
+            "description": "Indicates the partition kind is invalid. All Service Fabric enumerations have the invalid type. The value is zero."
+          },
+          {
+            "value": "Singleton",
+            "description": "Indicates that the partition is based on string names, and is a SingletonPartitionSchemeDescription object, The value is 1."
+          },
+          {
+            "value": "UniformInt64Range",
+            "description": "Indicates that the partition is based on Int64 key ranges, and is a UniformInt64RangePartitionSchemeDescription object. The value is 2."
+          },
+          {
+            "value": "Named",
+            "description": "Indicates that the partition is based on string names, and is a NamedPartitionSchemeDescription object. The value is 3"
+          }
+        ]
+      }
+    },
+    "PartitionSchemeDescription": {
+      "type": "object",
+      "discriminator": "partitionScheme",
+      "description": "Describes how the service is partitioned.",
+      "required": [
+        "partitionScheme"
+      ],
+      "properties": {
+        "partitionScheme": {
+          "$ref": "#/definitions/PartitionScheme",
+          "description": "Specifies how the service is partitioned."
+        }
+      }
+    },
+    "ProxyResource": {
```

Comment:
thanks for the feedback. Yes, you're right, removed

Reply:
@KrisBash I've actually reverted this change because we would need to make some changes to our service for this. Can we keep it like this for this release? we will add a new version soon with this changes